bpy_speckle/states/speckle_state.py

```diff
@@ -1,19 +1,19 @@
 import bpy
 from bpy.props import CollectionProperty, StringProperty, IntProperty, IntVectorProperty
+from bpy.types import PropertyGroup
 
-from ..ui.model_selection_dialog import speckle_model
 from ..ui.model_card import speckle_model_card
 
-class SpeckleState(bpy.types.PropertyGroup):    
-    ui_mode: StringProperty(name="UI Mode", default="NONE")
-    model_cards: CollectionProperty(type=speckle_model_card)
-    model_card_index: IntProperty(name="Model Card Index", default=0)
-    mouse_position: IntVectorProperty(size=2)
+class SpeckleState(PropertyGroup):
+    ui_mode: StringProperty(name="UI Mode", default="NONE")  # type: ignore
+    model_cards: CollectionProperty(type=speckle_model_card)  # type: ignore
+    model_card_index: IntProperty(name="Model Card Index", default=0)  # type: ignore
+    mouse_position: IntVectorProperty(size=2)  # type: ignore
```

Comment:
yep, exactly like this,
sucks that we have to do this, but I didn't find a better solution.